package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Design System developed for Wikia",
   "scripts": {
     "build": "npm run clean-build && npm run svgo-source && ember b -prod && cp -R style-guide/styles dist/scss && mv docs-dev/ docs &&  npm run svgo-dist",
+    "build-devbox": "ember b --environment=devbox",
```

Comment:
BTW I see it's already prepared to be one step process. Just PR description is wrong - see `npm run build-devbox` in `deploy-devbox.sh`